src/Npgsql/NpgsqlCommand.cs

```diff
@@ -832,7 +832,9 @@ void ValidateParameters(ConnectorTypeMapper typeMapper)
 
         void BeginSend(NpgsqlConnector connector)
         {
-            connector.WriteBuffer.Timeout = TimeSpan.FromSeconds(CommandTimeout);
+            connector.WriteBuffer.Timeout = CommandTimeout > 0
```

Comment:
Is this still necessary?

Reply:
You're right.
That's an oversight when reverting the changed timeout logic.

Reply:
Actually our timespans are an absolute mess.
I went ahead and annotated some of the values with documentation comments.